src/device-registry/utils/create-site.js

```diff
@@ -36,7 +35,13 @@ const {
 
 const createDeviceUtil = require("./create-device");
 
-const manageSite = {
+const { Kafka } = require("kafkajs");
+const kafka = new Kafka({
```

Comment:
Addressing a runtime for Kafka producer where things were undefined.
So brought all the configurations into each respective util to prevent this cyclic dependency runtime error.

Repetition of code but prevent this bug from happening ever again, hopefully!

@BenjaminSsempala @NoahNsimbe